composer.json

```diff
@@ -11,8 +11,8 @@
         }
     ],
     "require": {
-        "illuminate/contracts": "5.0 - 6.0",
-        "illuminate/support": "4.0 - 6.0"
+        "illuminate/contracts": "5.0 - 6.1",
+        "illuminate/support": "4.0 - 6.1"

```

Comment:
Should probably be something like:
```php
        "illuminate/contracts": "^5.0 - ^6.1",
        "illuminate/support": "^4.0 - ^6.1"
```

So that we don't have to update after each minor version.

Reply:
Better that way. Thanks

Reply:
My above suggestions was syntactically incorrect. Looks like it may be better as so:
```php
        "illuminate/contracts": "^5.0|^6.0",
        "illuminate/support": "^4.0|^5.0|^6.0",
```

Reply:
My preference goes to the last proposal!